packages/selfcare-client-users-updater/src/model/UsersEventPayload.ts

```diff
@@ -0,0 +1,27 @@
+import { z } from "zod";
+
+export const EventType = z.enum(["ADD", "UPDATE"]);
+export const RelationshipStatus = z.enum(["ACTIVE", "SUSPENDED", "DELETED"]);
+
+const UserEvent = z.object({
```

Comment:
```suggestion
const SCUser = z.object({
```
Let's use the same name that they use to define this type

Reply:
Applied in commit [4c12994](https://github.com/pagopa/interop-be-monorepo/pull/1789/commits/4c129946d852e8ce77934aaaddeb16be2934c172)